### PR TITLE
maint: add 'foreach-resource' command

### PR DESCRIPTION
**Pull request by praiskup**
This allows us to easily print info about resources, e.g. data field
(= typically IP address, if we speak about VMs).

### CODE REVIEW

**Comment by FrostyX** [comment]
This is really cool

File: bin/resalloc-maint [hunk 1]
```diff
@@ -45,6 +45,18 @@ p_t_list = subparsers.add_parser(
...
+foreach = subparsers.add_parser(
+    "foreach-resource",
+    help="Execute command for each resource",
+)
+foreach.add_argument(
+    "--command",
+    default="echo {name}",
+    help=(
+        "Shell command to execute.  This accepts variables like "
+        "{state}, {data_utf8}, {name}, etc."),
```
Comment (FrostyX) [inline bin/resalloc-maint:57]: Where do I find what "etc" means? :-)

Comment (praiskup) [reply bin/resalloc-maint:57]: Yeah, nowhere :-(

This functionality needs more care, and I didn't the needed time.  Better docs, more options.  Give me some time till I get back to this... (or feel free to add follow-up patches).

**Comment by praiskup** [comment]
Sorry, I don't have time to improve this now - and we are going to do the release now ... so merging

Feel free to submit patches (or I will, once there's some time).

**Pull request merged**